crates/bitwarden/src/tool/generators/password.rs

```diff
@@ -1,30 +1,73 @@
-use crate::error::Result;
+use std::collections::BTreeSet;
+
+use rand::{distributions::Distribution, seq::SliceRandom, RngCore};
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
-/// Password generator request. If all options are false, the default is to
+use crate::error::{Error, Result};
+
+/// Password generator request options. If all options are false, the default is to
 /// generate a password with:
 /// - lowercase
 /// - uppercase
 /// - numbers
 ///
 /// The default length is 16.
-#[derive(Serialize, Deserialize, Debug, JsonSchema, Default)]
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 #[cfg_attr(feature = "mobile", derive(uniffi::Record))]
 pub struct PasswordGeneratorRequest {
+    /// When set to true, the generated password will contain lowercase characters (a-z).
```

Comment:
This seems a bit verbose, what do you think about `Include lowercase characters (a-z)`?